django/applications/catmaid/control/importer.py

```diff
@@ -27,7 +28,7 @@
 
 from catmaid.models import (BrokenSlice, Class, Relation, ClassClass,
         ClassInstance, Project, ClassInstanceClassInstance, Stack, StackGroup,
-        StackStackGroup, ProjectStack, StackClassInstance, StackGroupRelation,
+        StackStackGroup, ProjectStack, StackClassInstance, StackGroupClassInstance, StackGroupRelation,
```

Comment:
This is a bugfix that seemed reasonable to make

Reply:
Yes, good catch! Linking class instances to stack groups during import doesn't seem to happened a lot recently. ;-)